tesla_powerwall/api.py

```diff
@@ -21,10 +19,7 @@ def __init__(
         verify_ssl: bool = False,
         disable_insecure_warning: bool = True,
```

Comment:
```suggestion
```

This is no longer needed then. Please also update the documentation in the `README.md` accordingly.